### ISSUE TITLE
remove leading symbol * in initial help

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#1828]
```m2
i1 : help

o1 = initial help
     ************

     Welcome to Macaulay2

     Try entering '2+2' at your next input prompt, which begins with i. The two output prompts begin with o.  The first one,
     with the equal sign, '=', gives the value computed from your input, and the second one, with the colon, ':', tells what
     type of thing the value is.

     Type one of these commands to get started reading the documentation:

       * copyright                         -- the copyright
       * help "Macaulay2"                  -- top node of the documentation.
       * help "reading the documentation"  -- 
       * help "getting started"            -- 
       * help "a first Macaulay2 session"  -- 
...
```
If the leading `*` is removed, then the entries can be entered directly when using Emacs. Currently, if you try the first one you get this:
```m2
i3 :        * copyright                         -- the copyright
stdio:3:8:(3): error: encountered invalid documentation tag: "Macaulay2, version 1.17.1
Copyright 2019 Daniel R. Grayson and Michael E. Stillman:
...
```

**Comment by DanGrayson** [comment]
good idea

**Comment by pzinn** [comment]
`help` produces a `Hypertext` object. if you do what you suggest, you should be careful to preserve a reasonable HTML structure (so it still looks OK in alternate output modes)

**Comment by mahrud** [comment]
I'm aware of how help works. Using PRE or CODE instead of UL suffices.

**Comment by pzinn** [comment]
CODE would be good, yes

**Comment by d-torrance** [comment]
Closed by #3272
